### PR TITLE
Remove home breadcrumb and add icons for claims/items

### PR DESCRIPTION
**Pull request by hobbitronics**
- Default to no home icon
- add umbrella and clipboard icons to items/claims
- code cleanup
image

### CODE REVIEW

File: src/pages/policies/[policyId]/claims.svelte [hunk 1]
```diff
@@ -36,7 +36,9 @@ const onGotoClaim = (event: CustomEvent<Claim>) => $goto(customerClaimDetails(ev
 </script>
 
 <Page layout="grid">
-  <Breadcrumb links={breadcrumbLinks} />
+  {#if isAdmin}
```
Comment (Schparky) [inline src/pages/policies/[policyId]/claims.svelte:39]: Why is this conditional? Does a customer not get breadcrumbs?

Comment (hobbitronics) [reply src/pages/policies/[policyId]/claims.svelte:39]: I isn't shown in the mockup and it would just be "claim"

Comment (hobbitronics) [reply src/pages/policies/[policyId]/claims.svelte:39]: Does the thumbs up indicate approved?

Comment (Schparky) [reply src/pages/policies/[policyId]/claims.svelte:39]: Yeah, I forgot I didn't approve the PR. Sorry about that.

**Review by Schparky — APPROVED** [review]

**Pull request merged**